geode-junit/src/main/java/org/apache/geode/internal/AvailablePortHelper.java

```diff
@@ -38,6 +34,23 @@
  * further calls to getRandomAvailablePort.
  */
 public class AvailablePortHelper {
+  static AtomicInteger currentMembershipPort;
+  static AtomicInteger currentAvailablePort;
+
+  static {
```

Comment:
Wouldn't this static block be better as a method? You could still invoke the method from the static block.

Overall, it looks like this class would benefit from changing to be a singleton (I hate to say it) but at least that would be cleaner from a design perspective than having static variables and static blocks.